src/processing/message_processor.go

```diff
@@ -0,0 +1,49 @@
+package processing
+
+import (
+	"BGPAlert/common"
+	"fmt"
+)
+
+func ProcessBGPMessages(msgChannel chan []common.BGPMessage, windowChannel chan []common.Window) {
+	var bucketMap = make(map[int64][]common.BGPMessage)
+
+	for bgpMessages := range msgChannel {
+		for _, msg := range bgpMessages {
+			timestamp := int64(msg.Timestamp)
+
+			// Round down the timestamp to the nearest multiple of 60 seconds
+			bucketTimestamp := (timestamp / 60) * 60
```

Comment:
I could be missing something but dividing something by 60 and then multiplying it by 60 looks a little suspicious 🤨